osrparse/replay.py

```diff
@@ -197,7 +197,7 @@ def pack_long(self, data):
 
     def pack_ULEB128(self, data):
         # https://github.com/mohanson/leb128
-        r, i = [], len(data)
```

Comment:
Let's do `data = data.encode("utf-8")` in `pack_string` instead. we're already writing `data.encode("utf-8")` in that method, so we should be calling `pack_ULEB128` on the encoded data at the higher level as well.

Reply:
sure